README.md

```diff
@@ -42,6 +42,12 @@ Alternatively, you can load via CDN with by adding a script tag to your app’s
 <script src="https://cdn.jsdelivr.net/npm/magic-sdk/dist/magic.js"></script>
 ```
 
+## 🔗 Testing
+Run test
+```bash
+yarn test
+```
+
```

Comment:
Let's add a bit more detail here:

Testing can also be on a per-package basis, like this:

```bash
PKG=magic-sdk yarn test
PKG=@magic-sdk/react-native yarn test
```

You can also glob the files you want to test specifically:

```bash
yarn test /test/**/constructor.spec.ts
```